kernels/optimized/cpu/op_mul.cpp

```diff
@@ -238,7 +130,9 @@ Tensor& opt_mul_out(
           out.numel());
     });
   } else if (selected_optimized_path != ElementwiseOptimizedPath::kNone) {
-    return handle_broadcast_mul(ctx, a, b, out, selected_optimized_path);
+    auto mul_lambda = [](auto x, auto y) { return x * y; };
+    return torch::executor::handle_broadcast_elementwise(
+        ctx, mul_lambda, a, b, out, selected_optimized_path);
```

Comment:
nit: mul_lambda is [`std::multiplies`](https://en.cppreference.com/w/cpp/utility/functional/multiplies), up to you which is more readable though